test/actionsmap/moulitest.yml

```diff
@@ -0,0 +1,42 @@
+
+#############################
+#     Global parameters     #
+#############################
+_global:
+    configuration:
+        authenticate:
+            - all
+        authenticator:
+            default:
+                vendor: dummy
+                help: Dummy Password
+            yoloswag:
+                vendor: dummy
+                help: Dummy Yoloswag Password
+
+#############################
+#       Test Actions        #
+#############################
+testauth:
+    actions:
+
+        none:
+            api: GET /test-auth/none
+            configuration:
+                authenticate: false
+
+        default:
+            api: GET /test-auth/default
+            configuration:
+                authenticate: all
+                authenticator: default
+
+#        only-api:
```

Comment:
Still needed?

Reply:
Hmmm yea idk, the intention was to have a test with `configuration: authenticate: api` where you need to authenticate to access the route, but only if from the API @.@